src/MarginTrading.Backend/Modules/BackendServicesModule.cs

```diff
@@ -97,6 +97,11 @@ protected override void Load(ContainerBuilder builder)
                 .As<IEventConsumer<OrderCancelledEventArgs>>()
                 .SingleInstance();
 
+            builder.RegisterType<VolumeEquivalentService>()
+                .As<IVolumeEquivalentService>()
+                .WithParameter(TypedParameter.From(_settings.EquivalentAssetSettings))
```

Comment:
I suggest to pass the whole `_settings` object to the constructor, and select the needed field inside of it.
This will remove the need of adding `WithParameter()` to the registration, make the module simpler and more stupid. It does not need to know the business logic of services configuration.
Also, this is the usual way of binding we use in this repo (and mt-mm as well).

Reply:
Fixed